zendesk/1.0/modules/ROOT/pages/zendesk-connector-reference.adoc

```diff
@@ -0,0 +1,749 @@
+* <<Config_BasicAuthentication, Basic Authentication Connection Provider>>
+* <<Config_Oauth2AccessCode, Oauth2 Access Code Connection Provider>>
+ | The connection types to provide to this configuration. | | x
+| Response Timeout a| Number |  The timeout for the request to the remote service. |  1 |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Response Timeout field |  MINUTES |
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Defines whether to send the request using streaming. If the value is set to `AUTO`, the best strategy is automatically determined based on the content of the request. |  AUTO |
+| Name a| String |  The identifier of this element used to reference it in other components |  | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
+|===
+
+==== Connection Types
+[[Config_ApiToken]]
+===== Api Token Connection Provider
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Default Headers a| Array of <<DefaultHeader>> |  |  |
+| Query Parameters a| Array of <<DefaultQueryParam>> |  |  |
+| Connection Timeout a| Number |  The timeout for establishing connections to the remote service |  30 |
+| Connection Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Connection Timeout |  SECONDS |
+| Use Persistent Connections a| Boolean |  If `false`, each connection will be closed after the first request is completed. |  true |
+| Max Connections a| Number |  The maximum number of outbound connections to keep open at the same time |  -1 |
+| Connection Idle Timeout a| Number |  A timeout for how long a connection can remain idle before it is closed |  30 |
+| Connection Idle Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the connection Idle Timeout |  SECONDS |
+| Proxy Config a| <<Proxy>> |  Reusable configuration element for outbound connections through a proxy |  |
+| Stream Response a| Boolean |  Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed and the body is streamed as it is received. When enabled, the response must eventually be read since, depending on the configured buffer size, the response might not fit into memory and processing stops until space is available. |  false |
```

Comment:
```suggestion
| Stream Response a| Boolean |  Whether or not to stream received responses, which means that processing continues as soon as all headers are parsed and the body is streamed as it is received. When enabled, the response must eventually be read since, depending on the configured buffer size, the response could not fit into memory and processing stops until space is available. |  false |
```

Reply:
"Might" fits better here.